lib/str.js

```diff
@@ -930,6 +931,17 @@ const Str = {
         return Constants.CONST.SMS.E164_REGEX.test(phone);
     },
 
+    /**
+     * Check for whether a phone number is valid.
+     * @param {String} phone
+     *
+     * @return {bool}
+     */
+
+    isValidPhoneNumber(phone) {
+        return parsePhoneNumber(this.removeSMSDomain(phone)).possible;
```

Comment:
I saw you already remove sms domain inside `ExpensifyMark`, do we really need to remove sms domain once again here?